elixir_server/test/elixir_server_test.exs

```diff
@@ -0,0 +1,108 @@
+defmodule ElixirServerTest do
+  use ExUnit.Case
+  use Plug.Test
+  doctest ElixirServer
+
+  test "greets the world" do
+    assert ElixirServer.hello() == :world
+  end
+
+  use ExUnit.Case
+
+  @options ElixirServer.Router.init([])
+
+  @rootDir File.cwd!()
+
+  @data %{
+    "options" => %{},
+    "paths" => [
+      "#{@rootDir}/test/resources/not_text.png",
+      "#{@rootDir}/test/resources/not_text.png",
```

Comment:
Why almost everywhere in this file are doubled paths?

Reply:
Just to test if it is working for same picture two times